src/events/messageCreateBump.ts

```diff
@@ -0,0 +1,22 @@
+import { ENV } from '@env';
+import type { BotEvent } from '@events';
+
+const TWO_HOURS = 7200000;
+
+/**
+ * @listensTo   - messageCreate
+ * @description - Emitted when a message is created.
+ */
+export const MESSAGE_BUMP: BotEvent = {
+    name: 'Message create bump',
+    kind: 'messageCreate',
+    once: false,
+    execute: (message) => {
+        if (!message.author.bot) return;
+        if (message.interaction && message.interaction.commandName == 'bump') {
```

Comment:
```suggestion
        if (message.interaction?.commandName == 'bump') {
```
You're not at work, you can use `?.` x)